skops/card/_model_card.py

```diff
@@ -1,18 +1,73 @@
 from __future__ import annotations
 
 import copy
+import json
 import re
 import shutil
 import tempfile
 from pathlib import Path
-from typing import Any
+from typing import Any, Union
 
 from modelcards import CardData, ModelCard
 from sklearn.utils import estimator_html_repr
 
 import skops
 
 
+def metadata_from_config(config_path: Union[str, Path]) -> CardData:
+    """Construct a ``CardData`` object from a ``config.json`` file.
+
+    Most information needed for the metadata section of a ``README.md``
+    file on Hugging Face Hub is included in the ``config.json`` file. This
+    utility function constructs a ``CardData`` object which can then be
+    passed to the :class:`~skops.card.Card` object.
+
+    This method populates the following attributes of the instance:
+
+    - ``library_name``: It needs to be ``sklearn`` for scikit-learn
+        compatible models.
+    - ``tags``: Set to a list, containing ``"sklearn"`` and the task of the
+        model. You can then add more tags to this list.
+    - ``widget``: It is populated with the example data to be used by the
+        widget component of the Hugging Face Hub widget, on the model's
+        repository page.
+
+    Parameters
+    ----------
+    config_path: str, or Path
+        Filepath to the ``config.json`` file, or the folder including that
+        file.
+
+    Returns
+    -------
+    card_data: ``CardData``
```

Comment:
Not sure about this suggestion:

```suggestion
    card_data: ``modelcards.CardData``
```

Reason being that users might not be familiar with `modelcards` and thus don't know where `CardData` comes from.